lib/settings/persistence/memory_settings_persistence.dart

```diff
@@ -0,0 +1,41 @@
+// Copyright 2022, the Flutter project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
```

Comment:
I am wondering do we actually have a LICENSE file?

Reply:
Removed